src/main/java/de/tum/in/www1/artemis/service/scheduled/ParticipantScoreScheduleService.java

```diff
@@ -112,6 +116,13 @@ public boolean isIdle() {
      */
     @PostConstruct
     public void startup() {
+        // The cypress tests require the "scheduling" profile for the messaging to work, but we don't want to run the scheduled tasks for participant scores during the tests as
+        // this causes flaky behaviour when deleting results.
+        // When the "cypress" profile is active, do not start up the ParticipantScoreScheduleService.
+        if (Set.of(environment.getActiveProfiles()).contains("cypress")) {
```

Comment:
This change is unacceptable. We cannot merge the PR with it!

Reply:
removed